src/github.com/getlantern/lantern-ui/app/js/app.js

```diff
@@ -122,15 +122,20 @@ var app = angular.module('app', [
           return;
         }
 
-        // try to reconnect indefinitely
-        // when the websocket closes
-        $interval(function() {
-          console.log("Trying to reconnect to disconnected websocket");
-          ds = $websocket('ws://' + document.location.host + '/data');
-          ds.onOpen(function(msg) {
-            $window.location.reload();
-          });
-        }, WS_RECONNECT_INTERVAL);
+        // Temporary workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1192773
+        if (navigator.userAgent.toLowerCase().indexOf('firefox') > -1) {
```

Comment:
I've experimented that after a while (between 40 and 50s), Firefox attempts to connect again and succeeds, in each one of the tabs. Will this prevent Firefox to ever reconnect?